downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -357,8 +370,8 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 - The `error<*>` syntax has been removed.
 - Relational expressions are no longer supported with numeric values when the static types of the operands belong to different ordered types.
 - The `lang.array:indexOf` and `lang.array:lastIndexOf` lang library functions cannot be used with values that do not belong to `anydata`.
-- An object used as the iterable value in a `foreach` statement, `from` clause, or `join` clause  must be a subtype of `object:Iterable`.
-- The `RawTemplate` type is now a distinct type.
+- An object used as the iterable value in a `foreach` statement, `from` clause, or `join` clause must be a subtype of `object:Iterable`.
```

Comment:
Why we are having breaking changes section only for language breaking changes? If we are having such a section shouldn't we mention all the breaking changes including stdlibs and tooling as well?

Reply:
@MaryamZi

Reply:
Ideally, we should. But IINM, most of the time they go under the main content itself for stdlib. Whereas for lang changes we can have them as bullet points.

Reply:
@MaryamZi - Then, shud this be a sub-topic under the language updates section?

Reply:
Yeah, we can do that too.

Reply:
Done.